pkg/scheduler/scheduler.go

```diff
@@ -333,17 +338,17 @@ func truncateMessage(message string) string {
 	return message[:max-len(suffix)] + suffix
 }
 
-func updatePod(client clientset.Interface, pod *v1.Pod, condition *v1.PodCondition, nominatedNode string) error {
+func updatePod(client clientset.Interface, pod *v1.Pod, condition *v1.PodCondition, nominatedNode *string) error {
```

Comment:
Document that `nil` nominatedNode means not to update.

Reply:
done.